scss/styles.scss

```diff
@@ -141,3 +121,65 @@
   padding: 10px;
   text-transform: uppercase;
 }
+
+.front-grid {
```

Comment:
you can use the `tech-container` class in both back and front to reuse the display grid and the grid-template-columns